models/sync_merkleroots.go

```diff
@@ -0,0 +1,38 @@
+package models
```

Comment:
hmmmm... we're using spv-wallet/models
do we need to make a copy of the structs from that package?

Reply:
Yes, but since spv-wallet is acting just as a "proxy" so to say in this case the actual structs were not created there. I believe they could stay here, we could rename the package to something different, but since spv-wallet is not concerned about the models maybe it's a good idea to keep them here? WDYT?

Reply:
after discussion this will be left as is here, and changed to use models from spv-wallet when merkleroot service is implemented on spv-wallet side